### ISSUE TITLE
Focus on chat window without toggling

### ISSUE THREAD

**Issue by hoangbn** [logancyang/obsidian-copilot#798]
**Is your feature request related to a problem? Please describe.**
Going back and forth between the notes and chat window is not seamless

**Describe the solution you'd like**
A hotkey to focus on the chat (from wherever), whether the chat is opened already or not

**Describe alternatives you've considered**
Use the toggle chat functionality, however, it only works well when the chat is already closed. Additionally, it re-indexes

**Comment by logancyang** [comment]
Dupe of #749 ? @zeroliu 

About re-index, you can turn it off in the setting with a different auto index strategy.

**Comment by zeroliu** [comment]
Yeah, the PR should fix the focus issue. But I also find a bug now that I'm testing it. The settings will reset after triggering the new open chat command. I think it's because we reuse the activate code that disables and re-enables the chat. It causes the settings to reset. I can take this as a follow-up ticket for #749.

**Comment by zeroliu** [comment]
https://github.com/logancyang/obsidian-copilot/pull/845 will be included in the next release. @hoangbn I suggest testing it again with "open window" command once it's live
